### PR TITLE
New package: dooble-2023.01.30

### LINKED ISSUE #39353: Package request: Dooble
### Package name

dooble

### Package homepage

https://textbrowser.github.io/dooble/

### Description

Dooble, the scientific browser. Minimal, cute, and unusually stable.

### Does the requested package meet the package requirements?

System, Compiled

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by Eloitor**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

closes #39353

### CODE REVIEW

File: srcpkgs/dooble/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="qt5-charts-devel qt5-webengine-devel qt5-webchannel-devel qt5-location-devel"
+short_desc="Dooble, the scientific browser. Minimal, cute, and unusually stable"
+maintainer="Eloi Torrents <elotor@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://textbrowser.github.io/dooble/"
+distfiles="https://github.com/textbrowser/dooble/archive/refs/tags/${version}.tar.gz"
+checksum=3af9a06c477f9317cf279389bd66237109fa205fb5526f4df7909250dc2c8e41
+
+do_install() {
+	vbin Dooble
```
Comment (classabbyamp) [inline srcpkgs/dooble/template:17]: based on the [AUR package](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dooble):
```suggestion
	vbin Dooble dooble
	vinstall dooble.desktop 644 usr/share/applications
	vinstall Icons/Logo/dooble.png 644 usr/share/pixmaps
	vlicense LICENSE
```
(renamed the bin file so it plays nice with the desktop file)

would be nice to also add the translations, but it seems to assume they're in `$PWD/Translations`. might be something to bring up with the upstream.
```
	vmkdir usr/share/dooble/Translations
	vcopy "Translations/dooble_*.qm" usr/share/dooble/Translations
```

Comment (Eloitor) [reply srcpkgs/dooble/template]: Thank you I'll take a look at it and open an issue upstream

Comment (sgn) [reply srcpkgs/dooble/template]: Patch this file: https://github.com/textbrowser/dooble/blob/94498e29d616de51287e752c57e287dcdccdc578/Source/dooble_application.cc#L81

Comment (sgn) [reply srcpkgs/dooble/template]: Here is the patch:
```diff
--- dooble_application.cc.orig	2022-09-23 20:15:31.021472658 +0700
+++ dooble_application.cc	2022-09-23 20:16:49.754946070 +0700
@@ -81,9 +81,7 @@
 
       if(path.isEmpty())
 	ok = m_translator->load("dooble_" + QLocale::system().name(),
-				QDir::currentPath() +
-				QDir::separator() +
-				"Translations");
+				"/usr/share/dooble/translations");
       else
 	ok = m_translator->load("dooble_" + QLocale::system().name(), path);
 

```

Comment (Eloitor) [reply srcpkgs/dooble/template]: Thank you!!!!!!!

Now only the aarch64 builds give problems...

Comment (classabbyamp) [reply srcpkgs/dooble/template]: don't forget to install the translations too
```
	vmkdir usr/share/dooble/Translations
	vcopy "Translations/dooble_*.qm" usr/share/dooble/translations
```

Comment (classabbyamp) [reply srcpkgs/dooble/template]: > Now only the aarch64 builds give problems...

add qt5-host-tools to hostmakedepends

File: srcpkgs/dooble/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'dooble'
+pkgname=dooble
+version=2022.06.15
+revision=1
+build_style=qmake
+configure_args="dooble.pro"
+hostmakedepends="qt5-qmake qt5-host-tools"
```
Comment (sgn) [inline srcpkgs/dooble/template:7]: Add `qt5-webengine` to `hostmakedepends`

**Comment by Eloitor** [comment]
This template now builds successfully.
I installed it in my computer, the translations appear in the correct folder `/usr/share/dooble/translations/`.
I then tried to change the language of the program in edit > settings > display > language but the only option available is English....

Everyting else seems to work fine.

**Comment by Eloitor** [comment]
I finally succeeded at patching Dooble to have translation files in `/usr/share/dooble/translations`.

**Pull request merged**